fedimint-escrow-client/src/cli.rs

```diff
@@ -57,18 +57,11 @@ pub(crate) async fn handle_cli_command(
             // finalize_and_submit txns to lock ecash by underfunding
             // how to call this method?
             let (operation_id, out_point, escrow_id) = escrow
-                .buyer_txn(
-                    Amount::from_sat(cost),
-                    buyer,
```

Comment:
Would name `buyer_pubkey` or `buyer_id` etc. to make it clear that we're passing around IDs.